java/apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -420,8 +410,7 @@
         </div>
    <% } %>
     <% if (!isIdentifierFirstLogin(inputType) && !isLoginHintAvailable(inputType)) { %>
-        <div class="field m-0">
-            <% if (!(StringUtils.equals(tenantForTheming, IdentityManagementEndpointConstants.SUPER_TENANT))) { %>
+            <div class="field m-0">
```

Comment:
Check wether there is an existing logic to switch the label between username /email